vendor/src/github.com/endophage/gotuf/signed/verify.go

```diff
@@ -94,15 +94,8 @@ func verifyMeta(s *data.Signed, role string, minVersion int) error {
 	return nil
 }
 
-var IsExpired = func(t string) bool {
```

Comment:
Simplifying IsExpired to allow us to have an easier way to change time in tests.